server/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java

```diff
@@ -36,6 +38,11 @@ public static class Path implements Writeable, ToXContentObject {
         long free = -1;
         long available = -1;
 
+        ByteSizeValue effectiveLowStage = null;
+        ByteSizeValue effectiveHighStage = null;
+        ByteSizeValue effectiveFloodStage = null;
+        ByteSizeValue effectiveFrozenFloodStage = null;
```

Comment:
naming nit/suggestion (it's hard to remember whether these things are the free bytes or the used bytes otherwise)

```suggestion
        ByteSizeValue lowStageFreeBytes = null;
        ByteSizeValue highStageFreeBytes = null;
        ByteSizeValue floodStageFreeBytes = null;
        ByteSizeValue frozenFloodStageFreeBytes = null;
```

Reply:
I realised when you did this that I didn't like the names I had suggested 🤦 see c5a72ea647a6660b7d8fead74fb199653fbd6e63 for my preference.